src/node/mod.rs

```diff
@@ -183,10 +217,7 @@ pub trait AudioNode {
     fn id(&self) -> &AudioNodeId {
         self.registration().id()
     }
-    fn channel_config_raw(&self) -> &ChannelConfig;
```

Comment:
Here's the original motivation of the PR:
- drop channel_config_raw
- drop channel_config_cloned
- only leave channel_config

While still allowing for users to apply constraints on the channel config values